src/api/server.rkt

```diff
@@ -153,6 +153,22 @@
          (match (server-action-action-type action)
            ['herbie-command
             (define command (server-action-associated-type action))
+            (match-define (herbie-command type test seed pcontext profile? timeline-disabled?)
+              command)
+            (match type
+              ['errors-hash
+               (when (hash-has-key? completed-work pcontext)
+                 (set! command
+                       (herbie-command 'errors
+                                       test
+                                       seed
+                                       (json->pcontext (hash-ref (hash-ref completed-work pcontext)
+                                                                 'points)
+                                                       (test-context test))
+                                       profile?
+                                       timeline-disabled?))
+                 (set! action (server-action 'herbie-command command)))]
+              [_ empty])
```

Comment:
This is a crazy design. Why is this a new type of job? Why not handle this when parsing the pcontext from JSON? Why do we even need a separate endpoint?

Separately, where's the error handling? We should gracefully inform Odyssey if the hash can't be found, say if the server got restarted while Odyssey was working. And Odyssey should ideally know how to handle this correctly, not sure what that would be (maybe re-submit without caching?).